LambdaEngine/Source/ECS/EntityRegistry.cpp

```diff
@@ -79,10 +79,7 @@ namespace LambdaEngine
 
 		EntityRegistryPage& topPage = m_EntityPages.top();
```

Comment:
```suggestion
		const EntityRegistryPage& topPage = m_EntityPages.top();
```
Could be const according to the linter.

Reply:
Fake news, linter's wrong. In line 81, a non-const member function is called:
`topPage.Pop(entity);`